.darglint

```diff
@@ -0,0 +1,10 @@
+# -*- mode: ini -*-
```

Comment:
Not a biggie, but can't you configure your editor locally to avoid having this kind of editor-dependent configuration here?

Reply:
This is really just to get syntax highlighting on github. Is there a different way to get syntax highlighting on gh for files with unknown extensions?

Reply:
Oh, I see. I didn't know about that. [git attributes](https://git-scm.com/docs/gitattributes) look like a very good match for this and it looks like they are supported (as well as both emacs local variables and VIM modelines):

https://github.com/github/linguist/blob/master/docs/overrides.md#using-gitattributes

I'm also fine if you want to leave it as it is, although I see an editor war coming... :boom: :laughing:

Reply:
Ok,  so we assign it to the first person that complains? (I mean you ;-))

Reply:
That's **exactly** why I said I was fine with it :smile:

Seriously, I'm also not convinced to go with the attributes because then it is *linguist*-dependent. There seem to be no perfect solution for this, so any solution is fine.